include/mysql/service_encryption.h

```diff
@@ -41,6 +41,7 @@ extern "C" {
 #endif
 #endif
 #endif
+#include <assert.h>
```

Comment:
Thank you for the review, that explains the unexpected noise I was seeing when trying to update the .pp files.

I was actually looking to use `DBUG_ASSERT`, but if I recall correctly, I wasn't able to include `my_dbug.h` since it wasn't in the include path this file's build. If that sounds correct and there's a reason for it, do you know of a better solution? (It's been a while since I've looked at this, so I might just be thinking of the wrong thing)

Reply:
No, you remember correctly. `DBUG_ASSERT` is what the server normally uses for asserts. But this here is a header to be included by plugins, and I'd rather avoid getting `my_dbug.h` (and everything it needs in turn) into plugins. So to minimize plugin contamination by server headers, I would personally use `assert()` here.

Reply:
That makes sense, thank you for the explanation.